### PR TITLE
theme designer: accessibility checker for one token pair

### PR DESCRIPTION
**Pull request by emmayjiang**
adds a working accessibility checker, currently only checks one pair but more will be added soon
image

### CODE REVIEW

File: packages/react-components/theme-designer/src/utils/color/shades.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+/* Calculates the contrast ratio between two colors. Used for verifying
+ * color pairs meet minimum accessibility requirements.
+ * See: https://www.w3.org/TR/WCAG20/ section 1.4.3
+ */
+export function getContrastRatio(color1: Vec3, color2: Vec3): number {
+  // Formula defined by: http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#contrast-ratiodef
+  // relative luminance: http://www.w3.org/TR/2008/REC-WCAG20-20081211/#relativeluminancedef
+
+  /* calculate the intermediate value needed to calculating relative luminance */
+  function _getThing(x: number): number {
```
Comment (sopranopillow) [inline packages/react-components/theme-designer/src/utils/color/shades.ts:12]: could this function be named `getIntermediateValue`?

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/acf1753ea566d89131058f5415961bfdafa2c742'>acf1753ea566d89131058f5415961bfdafa2c742</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by emmayjiang** [comment]
Added an example that shows the comparison of the colors.

image

**Review by micahgodbolt — APPROVED** [review]
[4:01 PM] Micah Godbolt
i'd suggest haveing the inaccessible pairs always open, and use the token name on the right most column

**Comment by emmayjiang** [comment]
image

**Pull request merged**